### PR TITLE
ac_search add "OR" filter

### PR DESCRIPTION
**Pull request by sktocha**

### CODE REVIEW

**Comment by leschenko** [comment]
can we have a test for this option?

**Comment by sktocha** [comment]
test + fix

File: lib/elasticsearch_autocomplete/model_addition.rb [hunk 1]
```diff
@@ -57,6 +57,14 @@ def ac_search(query, options={})
           end
 
           filter(:and, filters: options[:with].map { |k, v| {terms: {k => ElasticsearchAutocomplete.val_to_terms(v)}} }) if options[:with].present?
+
+          if options[:or].present?
+            or_filters = options[:or].map do |filter|
```
Comment (leschenko) [inline lib/elasticsearch_autocomplete/model_addition.rb]: may be `Array(options[:or])` to support `options[:or]` as Hash

Comment (sktocha) [inline lib/elasticsearch_autocomplete/model_addition.rb]: if will be array with one hash it's will be "with" filter.

Comment (leschenko) [inline lib/elasticsearch_autocomplete/model_addition.rb]:
```
          if options[:or].present?
            or_filters = Array(options[:or]).map do |filter|
```

Comment (sktocha) [inline lib/elasticsearch_autocomplete/model_addition.rb]: push -f last commit

**Pull request merged**